docs/deploy.md

```diff
@@ -7,17 +7,41 @@ description: The `sasjs deploy` command performs the actual deployment of a SASj
 sasjs deploy
 ====================
 
+## Prerequisites
+Before using this command, you will need to install the SASJS CLI and:  
+[add a deployment target](/add).  
+[compile services](/compile).  
+[build services](/build).
```

Comment:
perhaps these could be bullet points?

Reply:
instead of compile services / build services we should use:

* compile services / jobs
* build services / jobs

Reply:
For some reason, it doesn't change them to bullet, it just adds star when I tried